recipes/conda_gitenv/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: conda-gitenv
+  version: {{ version }}
+
+source:
+  fn: conda-gitenv-{{ version }}.tar.gz
+  url: https://github.com/SciTools/conda-gitenv/archive/v{{ version }}.tar.gz
+  sha256: afbcfea861c2fedecd640248a97f47b0538f29b7c3966eeaa724a7636d478633
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - gitpython
+    - pyyaml
+    - conda >=4.1.0
+    - conda-build-all
```

Comment:
Also would raise an issue in `conda-build-all` along the same lines as it should be doing this.

Sorry it's all kind of a mess as the solver can find ways around these pinnings. 😩